### PR TITLE
[Zerofox] Add format changes to malware endpoint and country entities

### PR DESCRIPTION
**Pull request by DNRRomero**
### Proposed changes

* Use pycountry to better annotate country codes
* add opencti_main_observable_type to indicators
* correct malware name to just be a file hash

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/2828

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Comment by DNRRomero** [comment]
Hi @helene-nguyen  is there anything left for getting a review in this PR?

**Comment by helene-nguyen** [comment]
@DNRRomero sorry for the delay, it is actually in review and you will have an update soon 😊

File: external-import/zerofox/src/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
+requests==2.31.0
```
Comment (Powlinett) [inline external-import/zerofox/src/requirements.txt]: on master `pycti` version will be 6.4.2 which will raise an error during `pip install`: 
image

I encourage you to loosen `requests` lib version as advised by `pip`

**Review by Powlinett — COMMENTED** [review]
Code looks good to me, I didn't see any errors during ingestion 👍

Note that this connector works by running it with Docker only, it doesn't handle setup with `config.yml` file. (@helene-nguyen @flavienSindou)

File: external-import/zerofox/src/requirements.txt (hunk 1 above)
Comment (dnrr-dev) [reply external-import/zerofox/src/requirements.txt]: Hi @Powlinett  thank you for the suggestion,
I loosened up the dependencies

regarding the config.yml, is there documentation on how to include that setup?

Comment (Powlinett) [reply external-import/zerofox/src/requirements.txt]: Thanks 🙂

You can find some documentation about `config.yml` [here](https://docs.opencti.io/latest/deployment/connectors/#parameters) and [an example](https://github.com/OpenCTI-Platform/connectors/blob/master/external-import/tenable-vuln-management/src/tenable_vuln_management/config_variables.py) of the way we handle config from both environment variables _and_ `config.yml` file.
It's useful if you want to [manually launch a connector](https://docs.opencti.io/latest/deployment/connectors/#manual-activation), for dev or debug purpose for instance.

But it's entirely up to you, we can still merge this PR without any further changes now, the intent of my comment was just to be sure that our team was aware of this "limitation".

Let us know 😇

**Review by Powlinett — APPROVED** [review]
No more dependencies issues 👍

**Comment by Powlinett** [comment]
Hi @DNRRomero, sorry about that but the latest release of pycti created a conflict in `requirements.txt` and I can't push the fix. Could you do it for us please? 🙏 You can keep your `requirements.txt` file (and optionally update pycti min version from 6.4.2 to 6.4.3)

I'll merge this PR right after, thanks a lot 🙏

**Comment by Powlinett** [comment]
Thanks @DNRRomero, please don't forget to sign your commits, I'll be pleased to merge your pull request today 😊

**Comment by DNRRomero** [comment]
I'm sorry @Powlinett  I'm having trouble signing my commits, I'll try again tomorrow

**Comment by DNRRomero** [comment]
Hi @Powlinett  I had trouble signing my commits, but @lderequesensS  is one of my colleagues and he signed the latest PR
Will that be okay?

**Comment by Megafredo** [comment]
Hi @DNRRomero, unfortunately not all commits are signed, so the merge is still blocked.

**Comment by lderequesensS** [comment]
Hi @Powlinett and @Megafredo, I squashed our changes and signed the commit 👍

**Pull request merged**